howdoi/howdoi.py

```diff
@@ -236,13 +271,25 @@ def _get_answer(args, links):
     text = text.strip()
     return text
 
+def _get_cachable_questions(query):
```

Comment:
Since this ultimately returns links, perhaps `_get_links_with_cache()` makes more sense.

Reply:
It replaces calls to _get_links + _get_questions in case of cache hit, so it's a direct rename. _get_links returns links, _get_questions returns questions ;)

Reply:
6abb8420a67cf0a65abc91fcd5b9462735aaa286